СРАВНИВАЕМ 4 ЧИСЛА
public class Solution
{
    public static void main(String[] args) throws Exception
    {
        //Напишите тут ваш код
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        String sn1 = reader.readLine();
        String sn2 = reader.readLine();
        String sn3 = reader.readLine();
        String sn4 = reader.readLine();
        int n1 = Integer.parseInt(sn1);
        int n2 = Integer.parseInt(sn2);
        int n3 = Integer.parseInt(sn3);
        int n4 = Integer.parseInt(sn4);

        if ((n1 > n2) && (n1 > n3)  && (n1>n4))
            System.out.println(n1);
        else if ((n2>n3) && (n2>n4))
        {
            System.out.println(n2);
        }
        else if (n3>n4)
        {
            System.out.println(n3);
        }
        else
        {
            System.out.println(n4);

    }
}


СЧИТЫВАЕМ СУММУ ВВЕДЕННУЮ С КОНСОЛИ
import java.io.BufferedReader;
import java.io.InputStreamReader;

public class Solution
{
    public static void main(String[] args) throws Exception
    {
        //Напишите тут ваш код

        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        int sum = 0;
        while (true)
        {
            String s = reader.readLine();

                if ("-1".equals(s))
                {
                    System.out.println(sum-1);
                    break;
                } else
                {
                    sum += Integer.parseInt(s);
                }


        }
    }
}

charsSogl.add(chars[i]);

Ввод числа в консоль int n = Integer.parseInt(reader.readLine());



Пока не запомнил как отрабатывать эксепшены, буду хранить тут шпору.

public class ExceptionExample2
{
public static void main(String[] args)
{
System.out.println("Program starting.");
try
{
System.out.println("Before method1 calling.");
method1();
System.out.println("After method1 calling. Never showed.");
}
catch (NullPointerException e)
{
System.out.println("Reference is null. Exception caught.");
}
catch (ArithmeticException e)
{
System.out.println("Divided by zero. Exception caught.");
}
catch (Exception e)
{
System.out.println("Any other errors. Exception caught.");
}
System.out.println("Program still running.");
}
public static void method1()
{
int a = 100;
int b = 0;
System.out.println(a / b);
}
}



копируем данные из 1 файла в другой

public class Solution
{
    public static void main(String[] args) throws IOException
    {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));

        String sourceFileName = reader.readLine();
        String destinationFileName = reader.readLine();

        FileInputStream fileInputStream = new FileInputStream(sourceFileName);
        FileOutputStream fileOutputStream = new FileOutputStream(destinationFileName);

        int count = 0;
        while (fileInputStream.available()>0)
        {
            int data = fileInputStream.read();
            fileOutputStream.write(data);
            count++;
        }

        System.out.println("Скопировано байт " + count);

        fileInputStream.close();
        fileOutputStream.close();
    }
}